internal/operators/ocs/utils.go

```diff
@@ -35,15 +35,16 @@ spec:
 }
 
 type storageInfo struct {
-	OCSDisks int
+	OCSDisks int64
 }
 
 func OCSStorageCluster(StorageClusterManifest string, ocsDiskCounts int64) (string, error) {
 
 	// currently we are supporting only single storage set
 	// TODO support storage set according to disk count
+	ocsDiskCounts /= 3
```

Comment:
The comment above is no longer relevant?

Reply:
fixed